src/r_things.c

```diff
@@ -689,6 +689,20 @@ static void R_ProjectSprite (mobj_t* thing)
       vis->colormap[1] = fullcolormap;
     }
 
+  // ID24 per-frame tranmap
```

Comment:
Let's use "per-tick" instead of "per-frame." In the original Doom, frames and ticks were the same, but with interpolated uncapped rendering, that's no longer true.